src/alerts/flash-refactor.scss

```diff
@@ -0,0 +1,133 @@
+// stylelint-disable selector-max-type
+
+// Flash alert
+
+.Flash {
```

Comment:
After chatting with @ashygee, we agreed to rename this component to "Alert".

Additionally, @alliethu gave some great insights into how to handle accessibility for it:

> A couple alert notes:
> * Use `role="alert"` by default
>   * If more than one alert can display at once, use `role="status"` so the screen reader doesn’t interrupt itself with the latest message.
> * Provide a cue for the alert type (info, warning, error, success)
>   * Visually with an icon
>   * Audibly using visually hidden text that precedes the alert
>   * If no dismiss/close button, the alert must be present for a minimum of 5 seconds before dismissing.
> * Connect title and description using `aria-describedby`

Reply:
@vdepizzol based on the conversation with @langermank I think that "Banner" is a better name for this component. If we do decide to keep "Alert" though we may want to clarify the difference between this and a `Toast` notification/alert. I also think we may want to audit the usage of the `Toast` since this alert styling is much more common across the patterns seen within the GitHub experience.

Reply:
@ashygee just renamed the component to Banner.

I haven't updated the stories with WAI-ARIA considerations because I think these are still in flux. Per instance, there are many existing [scenarios](https://www.figma.com/file/ngdfD1BrM9tpRbbdeBUW18/Pattern%3A-Messaging-banners?node-id=0%3A1)[^1] where `role="alert"` shouldn't be useful. I think we can polish the accessibility definitions directly in the PVC updates.

[^1]: Link only available to hubbers.